README.md

```diff
@@ -64,15 +65,11 @@ Traefik routes requests based on host name. which includes the coinstack name. F
 
 ## Docker-Compose Local Dev Instructions
 
-### _Lightweight local development environment_
-
----
-
-#### **Prerequisites**
+#### Prerequisites
 
 - Install [docker-compose](https://docs.docker.com/compose/install/)
 
-#### **Running**
+#### Running
 
 - `unchained/node/packages/client` generates a typescript client for each coinstack using the generated `swagger.json` files. To ensure everything successfully builds, make the golang coinstacks first:
```

Comment:
this section is no longer relevant if you want to remove while you are at it

Reply:
https://github.com/shapeshift/unchained/pull/531/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R61 is also not exactly accurate or relevant, but could add first `Running` step as:

- Install node dependencies
```
cd node && yarn
```